### PR TITLE
Update README with installation for repeatable scripts

### PR DESCRIPTION
**Pull request by lawrencegripper**
Hi, 

I've had a go at adding some install instructions for pinning to a version of `azcopy` for use in scripts. How does it look? Hopefully useful to others, let me know if you'd like any edits/changes. 

Related to #253

### CODE REVIEW

**Comment by zezha-msft** [comment]
Thanks @lawrencegripper!

@seguler could you please review the doc change?

**Comment by JohnRusk** [comment]
Hi Lawrence,

I see this PR has slipped though the cracks.  I'll discuss it with the rest of the team later in the week. (I know we didn't want to encourage pinning before GA.  Now that we are GA, I want to check in with the rest of the team to confirm the extent to which we are comfortable with pinning.  I agree with your point that folks want to be able to script against a predictable version).

File: README.md [hunk 1]
```diff
@@ -49,6 +49,32 @@ cd .\azcopy_windows_amd64_10.0.8
 .\azcopy.exe
 ```
 
+### Installation for repeatable scripts
+
+The version of `azcopy` available at `https://aka.ms/downloadazcopy-v10-linux` will be updated over time. If you plan on using `azcopy` as part of a script it's best to pin to a known version so future updates don't affect your scripts. 
+
+To pin to a version of `azcopy`
+
+1. Run `curl -v https://aka.ms/downloadazcopy-v10-linux` and review the output to locate the current version URL. Here is an example output from `curl` showing the `301 Moved Permanently` response.
```
Comment (JohnRusk) [inline README.md]: Windows equivalent is this PowerShell command:
` (curl https://aka.ms/downloadazcopy-v10-windows -MaximumRedirection 0 -ErrorAction silentlycontinue).RawContent`

**Comment by JohnRusk** [comment]
@lawrencegripper I had a chat with the team, and yes, we think your PR is a good idea and we should go ahead with it.  

One question: before we merge the PR, might you be able to update it so that, in addition to the Linux commands, it also shows Windows commands?  E.g. to find the versioned URL this works in PowerShell 

` (curl https://aka.ms/downloadazcopy-v10-windows -MaximumRedirection 0 -ErrorAction silentlycontinue).RawContent`

File: README.md (hunk 1 above)
Comment (lawrencegripper) [reply README.md]: I've added this one in now - good call

**Comment by lawrencegripper** [comment]
@JohnRusk no problem, I've attempted to update the script but I can't find any universal way to do the equivalent of `--strip-compontents=1` from `tar` in windows. 

Do you know of a way to do this with `Expand-Archive` - [docs here](https://docs.microsoft.com/en-us/powershell/module/microsoft.powershell.archive/Expand-Archive?view=powershell-6).

My suggestion would be that the packages releases don't put the exe in the zip under a versioned path and this is why this step is required. If the zip path didn't contain the version is then things would be simpler.

**Comment by JohnRusk** [comment]
Hi @lawrencegripper. Sorry about the delayed reply.  I don't know of any way to replicate --strip-components directly. But, you could make a child folder with a unique name (e.g. from a guid) so that it won't clash with anything else. Expand-Archive into that child folder.  Cd into there then cd az* (or whatever will take you into the version AzCopy folder that's inside the zip), then copy * ..\.., then cd ..\.., then delete the child directory.

**Comment by zezha-msft** [comment]
Since our README has changed dramatically, @normesta could you please incorporate this PR into the docs website instead?

**Comment by normesta** [comment]
Yep. I'll have a look at this one. Thx!

**Comment by lawrencegripper** [comment]
Cool, I'll close this PR now then.

**Comment by normesta** [comment]
Hi all - I'm a bit confused about the scenario here. It seems logical to want to pin a script to a specific version of AzCopy, but these steps don't seem to be doing that. I'm sure I'm missing something here, but these steps appear to use a PowerShell command to obtain the latest URL of the script. That would by definition be the latest updated version of the script right? That's exactly what we wouldn't want to do. Can you please explain how this gives the developer / administrator the URL of a specific version and not just the latest version which is what it seems like we're getting here? Thx!

**Comment by JohnRusk** [comment]
Norm, there are two parts to the scripts. The first part, as you have seen, discovereds the underlying URL of the latest version. So you have a fixed URL for that specific version. The second part is a download script … that uses that specific URL.  So the second part is pinned to the version.

(whereas if you write a download script that uses the aka.ms link, it's not pinned to a version).

**Comment by normesta** [comment]
Thank you. Yes this sequence makes sense. Finding the current version happens once. Then, for each use of the script downloads that specific version of AzCopy. Why not just store a version of AzCopy for use with a script. Why always download AzCopy as part of running the script?

**Comment by zezha-msft** [comment]
@normesta in a typical CI environment (like Travis), environments are completely wiped after a run and new jobs start from brand new environments, thus they have to download AzCopy every time.

**Pull request closed without merging**